products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+releasePolicyLink: https://www.sqlite.org/news.html
+changelogTemplate: https://sqlite.org/releaselog/__LATEST__/.html
+activeSupportColumn: false
+eolColumn: Support Status
+versionCommand: sqlite3 --version
+releaseDateColumn: true
+auto:
+-   git: https://github.com/sqlite/sqlite.git
+  # https://rubular.com/r/KlemgnguNe0e5X
+  #  regex: ^REL_?(?<major>[1-9]\d*)_(?<minor>0|[1-9]\d*)_?(?<patch>\d+)?$
+releases:
+-   releaseCycle: "3.40"
```

Comment:
Are multiple minor versions of the same major supported in parallel ? If no, `3` cycles should be collapsed.

Reply:
According to their [LTS Policies](https://www.sqlite.org/lts.html), I may say that they do : do you share the same analysis ?

Reply:
See https://github.com/endoflife-date/endoflife.date/pull/2442#discussion_r1097150292

Reply:
Versions collapsed and description updated.